pkg/istio-agent/xds_proxy.go

```diff
@@ -655,6 +655,7 @@ func (p *XdsProxy) buildUpstreamClientDialOpts(sa *Agent) ([]grpc.DialOption, er
 	dialOptions := []grpc.DialOption{
 		tlsOpts,
 		keepaliveOption, initialWindowSizeOption, initialConnWindowSizeOption, msgSizeOption,
+		grpc.WithBlock,
```

Comment:
It was there In the original pr, can you checkout when and why this is removed. I can't remember

Reply:
I looked at the history but could not find it in the original file also.

Reply:
#26838